### ISSUE TITLE
Setup Question

### ISSUE THREAD

**Issue by damienbutt** [tonarino/innernet#130]
Hi all,

I have read through the docs but I am a little confused setting this up.

For my network name I went with my domain name. "domain.com"
Network CIDR I went with the default 10.42.0.0/16.
Yes to auto-filling my public IP from 1.1.1.1 which was completed successfully.

The external endpoint was the bit that confused me. What should that be? Obviously not my public IP as that was auto-filled from the previous step. So is the external endpoint the IP address of the machine I am running the server on? That was I did, as a guess. "192.168.10.84:51820" 

Then the listen port default was 51820 which I went with. 

The setup finished but I have not yet started the service. Can anyone advise about the external endpoint and if I have that correct? If it's not the machine the server is running on, what should it be?

Thanks,

D

**Comment by afics** [comment]
The external endpoint is the one outside the VPN i.e. the one innernet managed wireguard tunnels connect to. This is usually a public IP. The internal endpoint is the one the innernet clients connect to over the wireguard tunnel, usually the very first address in your network CIDR.

Regards,
afics

**Comment by damienbutt** [comment]
Right. What is the function of the "auto-fill public IP" step in the setup supposed to do then? I think that's what confused me a bit. Was it supposed to auto-fill the external endpoint address with my public IP? That didn't happen but the little tick on the auto-fill step was green and it said it was completed. 

So the external endpoint should be <Public IP>:51820? 

Is there a way to edit bits of the server config? Or do I need to delete and re-create the server?

Thanks,

D

**Comment by afics** [comment]
> Was it supposed to auto-fill the external endpoint address with my public IP?

Yes, interesting that this did not work for you.

> So the external endpoint should be "Public IP":51820?

Yes.

> Is there a way to edit bits of the server config? Or do I need to delete and re-create the server?

Recreation is a possibility. As far as I know your only other option is to change it directly in the innernet server sqlite database, usually under `/var/lib/innernet-server/something.db`.

Regards,
Armin

**Comment by damienbutt** [comment]
OK. Thanks.

I'm running the server on a MacMini. Or at least trying to. Although there are instructions on how to install with Homebrew etc, the starting/stopping of the server instructions uses the systemd (Linux) syntax.  I have tried using `brew services start innernet@server-name` to invoke launchctl but no luck. I'm guessing because there is no `.plist` file?
